lib/Zonemaster/Backend/DB/MySQL.pm

```diff
@@ -95,6 +95,92 @@ sub dbh {
     return $self->dbhandle;
 }
 
+sub create_db {
```

Comment:
Maybe include the `create_db` method in the `requires` of DB.pm ?

Reply:
done